src/service/resource.js

```diff
@@ -1115,6 +1118,9 @@ export class Resource {
   unload() {
     this.pause();
     this.unlayout();
+    if (this.element.V1()) {
+      this.element.unmount();
```

Comment:
This will cause `pause` itself.

Reply:
Correct. Basically for V1 `Resource.unload` becomes `Element.unmount` and otherwise things are very similar. On unmount we also have to pause.

Reply:
Refactored it a bit. Only one pause is called now via `unmount()`.